internal/connections_test.go

```diff
@@ -30,7 +30,6 @@ func TestAccConnections(t *testing.T) {
 	})
 
 	connUpdate, err := w.Connections.Update(ctx, catalog.UpdateConnection{
-		Name:    connCreate.Name,
```

Comment:
we don't need this

Reply:
Shouldn't we be removing `NameArg` instead of `Name`?

Reply:
I guess that change didn't make it through...